api/src/main/kotlin/com/gmtkgamejam/plugins/PostRoutes.kt

```diff
@@ -15,7 +19,33 @@ fun Application.configurePostRouting() {
     routing {
         route("/posts") {
             get {
-                call.respond(service.getPosts())
+                val params = call.parameters
+
+                // All Posts found should be active
+                // TODO: Handle deleted posts differently for admins (if at all)
+                val filters = mutableListOf(PostItem::deletedAt eq null)
+
+                params["description"]?.split(',')
+                    // The regex is the easiest way to check if a description contains a given substring
```

Comment:
For completeness: `contains` only works for iterables, and there was no magic I could find to parse the string field into a string[] for the query